site/gulpfile.js

```diff
@@ -0,0 +1,115 @@
+'use strict';
```

Comment:
Putting a comment here because I can't in `./site/debug.log` ... `debug.log needs to be ignored and thrown out.